beets/ui/commands.py

```diff
@@ -1124,24 +1129,25 @@ def update_items(lib, query, album, move, pretend):
                     item._dirty.discard(u'albumartist')
 
             # Check for and display changes.
-            changed = ui.show_model_changes(item,
-                                            fields=library.Item._media_fields)
+            changed = ui.show_model_changes(
+                item,
+                fields=fields or library.Item._media_fields)
 
             # Save changes.
             if not pretend:
                 if changed:
                     # Move the item if it's in the library.
                     if move and lib.directory in ancestry(item.path):
-                        item.move()
+                        item.move(fields_to_store=fields)
```

Comment:
Because we're *about* to call `item.store`, maybe the right thing to do here is just to disable the redundant store to the database inside `item.move`.